docs/chapter_computational_complexity/time_complexity.md

```diff
@@ -79,13 +79,31 @@ $$
 === "JavaScript"
 
     ```js title=""
-
+    // 在某运行平台下
+    function algorithm(n){
+        let a = 2 // 1 ns
+        a = a + 1 // 1 ns
+        a = a * 2 // 10 ns
+        // 循环 n 次
+        for(let i = 0; i < n; i++){ // 1 ns
+            a = a + i // 5 ns
+        }
+    }
     ```
 
 === "TypeScript"
 
     ```typescript title=""
-
+    // 在某运行平台下
+    function algorithm(n: number): void {
+      let a = 2 // 1 ns
```

Comment:
Four spaces is required for the indentation in this project.

Reply:
I have fixed it.